web-admin/src/routes/+layout.ts

```diff
@@ -75,10 +76,16 @@ export const load = async ({ params, url, route }) => {
 
   // Get organization permissions
   let organizationPermissions: V1OrganizationPermissions = {};
+  let organizationLogoUrl: string | undefined = undefined;
   if (organization && !token) {
     try {
-      organizationPermissions =
-        await fetchOrganizationPermissions(organization);
+      const organizationResp = await queryClient.fetchQuery({
+        queryKey: getAdminServiceGetOrganizationQueryKey(organization),
+        queryFn: () => adminServiceGetOrganization(organization),
+        staleTime: Infinity,
+      });
```

Comment:
Nit/idea: if interested in abstracting out the guts of a `fetchQuery` call like this one, we could factor out the `queryOptions` into a separate file, like [this blog post](https://tkdodo.eu/blog/the-query-options-api) advocates

Reply:
Remind me why `staleTime: Infinity` is needed?

Reply:
Every switch of org fetches the org details otherwise. Seems redundant to me. There are a few more of those we should probably clamp on.